File: src/main/java/com/sup/dev/java_pc/tools/ToolsPcScreen.java
package com.sup.dev.java_pc.tools;

import java.awt.*;

public class ToolsPcScreen {


    public static double getScreenWidth(){
        return Toolkit.getDefaultToolkit().getScreenSize().getWidth();
    }

    public static double getScreenHeight(){
        return Toolkit.getDefaultToolkit().getScreenSize().getHeight();
    }

    public static double getScreenDPI(){
        return 240;
    }

    public static double getScreenDPRation(){
        return getScreenDPI()/160;
    }

    public static double getScreenSPRation(){
        return getScreenDPI()/160;
    }

}

File: src/main/java/com/sup/dev/java_pc/tools/ToolsPcScreen.kt
package com.sup.dev.java_pc.tools

import java.awt.Toolkit


object ToolsPcScreen {


    val screenWidth: Double
        get() = Toolkit.getDefaultToolkit().screenSize.getWidth()

    val screenHeight: Double
        get() = Toolkit.getDefaultToolkit().screenSize.getHeight()

    val screenDPI: Double
        get() = 240.0

    val screenDPRation: Double
        get() = screenDPI / 160

    val screenSPRation: Double
        get() = screenDPI / 160

}
